CONTRIBUTING.md

```diff
@@ -255,6 +255,13 @@ jhipster --with-entities
 
 You should see your changes reflected in the generated project.
 
+Note: The generated project might not build properly in case the generator is using a
+snapshot version of [jhipster/jhipster](https://github.com/jhipster/jhipster). This issue is mentioned in; https://github.com/jhipster/generator-jhipster/issues/9571. In
+this case clone the jhipster/jhipster project and build it using: 
+```
+./mvnw clean install -Dgpg.skip=true` 
```

Comment:
can you remove the last char `

Reply:
@pascalgrimaud : Good catch. Done :smile: